learn/experimental/vector_search.mdx

```diff
@@ -30,39 +30,139 @@ curl \
   }'
 ```
 
-### Generate vector embeddings using a third-party tool
+### Generate vector embeddings
 
-To use vector search, you must generate embeddings with a third-party tool such as [OpenAI](https://openai.com/), [HuggingFace](https://huggingface.co/), and [Cohere.AI](https://cohere.com/).
+To use vector search, first configure the `embedders` index setting. You may configure multiple embedders for an index. 
 
-<Capsule intent="note" title="What is an embedding?">
-Vector search relies on embeddings. Vector embeddings are a way of representing data that identify similarities in meaning and context.
+Embedders generate vector data from your documents. Meilisearch natively supports [OpenAI](https://openai.com/) and [HuggingFace](https://huggingface.co/) embedders. 
+
+It is also possible to use custom embedders. In this case, you must generate the embeddings manually and add include them as a field in your documents.
+
+#### Generate auto-embeddings with OpenAI 
+
+Use the `embedders` index setting of the [update `/settings` endpoint](/reference/api/settings) to configure one or more embedders for an index:
+
+```sh
+curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "embedders": {
+      "default": {
+        "source":  "openAI",
+        "apiKey": "anOpenAiApiKey",
```

Comment:
I think we should specify how to procure a key + specify that for good indexing performance it is recommended to use a key of tier 2 or above

Reply:
Definitely yes on the tier 2 bit. I'll try to give slightly more directions on procuring the API key, but I really don't want to get into documenting openAI directly.